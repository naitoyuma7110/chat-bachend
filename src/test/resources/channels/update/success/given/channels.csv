id,name
1, はじめてのチャンネル
2, 2つ目のチャンネル