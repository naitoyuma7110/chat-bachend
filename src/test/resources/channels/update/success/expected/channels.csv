id,name
1,updateしたレコード
2,テストデータ